index.js

```diff
@@ -1,60 +1,69 @@
 'use strict'
 
 const { Client } = require('discord-rpc');
+const julian = require('julian');
+const moment = require('moment');
 
-const clientId = '811111315988283413';
-
+const clientId = '811145575134658561';
 const day = 24 * 60 * 60;
 const landingTimestamp = 1613681692;
 
+function getMSD(earthTime) {
+    earthTime = moment(earthTime).toDate();
+    return (julian(earthTime) - 2405522.0028779) / 1.0274912517;
+};
+function getMissionSol() {
+    let date = new Date().getTime();
+
+    return Math.floor(getMSD(date) - getMSD(new Date(1613681692 * 1000).toISOString()));
+}
```

Comment:
```suggestion
function getMissionSol() {
  const date = Date.now();
  return Math.floor(getMSD(date) - getMSD(new Date(landingTimestamp * 1000).getTime()));
}
```
Pass a timestamp here